tests/release/const.go

```diff
@@ -28,9 +50,16 @@ const (
 	releasePlanAdmissionCreationTimeout   = 1 * time.Minute
 	releaseCreationTimeout                = 1 * time.Minute
 	releasePipelineRunCreationTimeout     = 5 * time.Minute
-	releasePipelineRunCompletionTimeout   = 10 * time.Minute
+	releasePipelineRunCompletionTimeout   = 20 * time.Minute
 	avgControllerQueryTimeout             = 1 * time.Minute
 	pipelineServiceAccountCreationTimeout = 3 * time.Minute
+	oneMinuteTimeout                      = 1 * time.Minute
```

Comment:
this is used for a service account creation. Shouldn't it be named accordingly, much like the others are named after what resource they are used for?

Reply:
fixed